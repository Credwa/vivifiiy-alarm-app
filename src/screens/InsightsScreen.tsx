import Background from '@/components/Background';
import VivText from '@/components/VivText';
import Colors from '@/constants/Colors';
import React from 'react';
import { StyleSheet, View } from 'react-native';
import { SafeAreaView } from 'react-native-safe-area-context';

export default function InsightsTabScreen() {
  return (
    <Background>
      <SafeAreaView style={styles.container}>
        <View style={styles.header}>
          <VivText fontName="Title4" color={Colors.greyLight1}>
            Insights
          </VivText>
        </View>
      </SafeAreaView>
    </Background>
  );
}
const styles = StyleSheet.create({
  container: {
    flex: 1,
    marginTop: 20
  },
  header: {
    flexDirection: 'row',
    height: 44,
    marginBottom: 25,
    alignItems: 'center',
    justifyContent: 'space-between',
    alignContent: 'space-between'
  }
});
