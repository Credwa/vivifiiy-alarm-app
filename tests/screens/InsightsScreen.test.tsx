import InsightsScreen from '@/screens/InsightsScreen.tsx';
import { Ionicons } from '@expo/vector-icons';
import * as Font from 'expo-font';
import React from 'react';
import renderer from 'react-test-renderer';

describe('<InsightsScreen />', () => {
  beforeEach(async () => {
    await Font.loadAsync({
      ...Ionicons.font
    });
  });

  it(`renders correctly`, () => {
    const tree = renderer.create(<InsightsScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
