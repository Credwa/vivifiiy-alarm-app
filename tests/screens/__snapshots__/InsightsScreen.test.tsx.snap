// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InsightsScreen /> renders correctly 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "flex": 1,
      "paddingHorizontal": 20,
      "resizeMode": "cover",
    }
  }
>
  <Image
    source={1}
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": undefined,
          "width": undefined,
        },
        undefined,
      ]
    }
  />
  <RNCSafeAreaView
    style={
      Object {
        "flex": 1,
        "marginTop": 20,
      }
    }
  >
    <View
      style={
        Object {
          "alignContent": "space-between",
          "alignItems": "center",
          "flexDirection": "row",
          "height": 44,
          "justifyContent": "space-between",
          "marginBottom": 25,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#869DB2",
            },
            Object {
              "fontFamily": "Nunito_400Regular",
              "fontSize": 28,
            },
            Object {
              "color": "#CEE1F2",
            },
            undefined,
          ]
        }
      >
        Insights
      </Text>
    </View>
  </RNCSafeAreaView>
</View>
`;
